lib/Macros/Sources/SwiftMacros/DebugDescriptionMacro.swift

```diff
@@ -473,6 +482,28 @@ extension String {
   }
 }
 
+extension String {
+  fileprivate func escapedForLLDB() -> String {
+    guard #available(macOS 13, *) else {
```

Comment:
We don't actually need this fallback, right? This will never run on pre-macOS 13, unless I'm missing something.

Reply:
I agree that it should not be needed. `build-script` compiles with a deployment target <13. Do you have a suggestion how to avoid this?

Reply:
Leave it for now